freqtrade/freqtradebot.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
 
         self.rpc: RPCManager = RPCManager(self)
 
-        exchange_name = self.config.get('exchange', {}).get('name', 'bittrex').title()
+        exchange_name = self.config.get('exchange', {}).get('name').title()
```

Comment:
Using the default here was pointless anyway, since it was in the "required" fields and not having it wouldn't have passed validation...

Reply:
Excactly, That's an illustration of the negative impact when the default values are scattered as caviar throughout the code. Plus, the tests may test different things than they are supposed to test.